packages/list/list.scss

```diff
@@ -0,0 +1,86 @@
+@import "~@fremtind/jkl-core/variables/_all.scss";
+@import "~@fremtind/jkl-core/_functions.scss";
+
+$list-indent: $component-spacing--xl;
+$item-space-between: $component-spacing--large;
+$item-indent: $component-spacing--xxl;
+
+$diamond: "\25C6";
+$diamond-hollow: "\25C7";
+$square: "\25A0";
+
+.jkl-list {
+    list-style: none;
+    padding: 0;
+    margin: 0;
+    padding-left: $list-indent;
+
+    & > .jkl-list__item {
+        margin: $item-space-between 0;
+        padding-left: $item-indent;
+
+        &::before {
+            position: absolute;
+            margin-left: -3ch;
+        }
+
+        &:first-of-type {
+            margin-top: 0;
+        }
+        &:last-of-type {
+            margin-bottom: 0;
+        }
+    }
+    & & > .jkl-list__item {
+        padding-left: $item-indent;
+        &:first-of-type {
+            margin-top: $item-space-between;
+        }
+        &:last-of-type {
+            margin-bottom: $item-space-between;
+        }
+    }
+
+    &--bullet {
+        & > .jkl-list__item {
+            &::before {
+                content: $diamond;
+            }
+        }
+        & & > .jkl-list__item {
+            &::before {
+                content: $diamond-hollow;
+            }
+        }
+        & & & > .jkl-list__item {
+            &::before {
+                content: $square;
+            }
+        }
+    }
```

Comment:
Trenger dette å være en egen modifier? Siden vi setter `list-style` til `none` i rotklassen ville jeg definert et standardutseende på rot også, så er man sikker på at man ikke ender opp med en liste som ikke ser ut som en liste. Eventuelt sette `list-style: none` inne i variantene i stedet for på rot.

**Edit**: Hvis det skal fortsette å være en påkrevd variant (`.jkl-list--bullet`) må det også endres i `markdownRenderer.ts` i portalen. Den rendrer lister med kun `jkl-list` som klasse slik det er nå.